server/services/flood.js

```diff
@@ -42,9 +42,14 @@ module.exports = {
     return util.getJson(`${serviceUrl}/floods`)
   },
 
-  // bbox refers to binding box defined using coordinates x1,y1 & x2, y2
   getFloodsWithin (bbox) {
-    return util.getJson(`${serviceUrl}/floods-within/${bbox[0]}/${bbox[1]}/${bbox[2]}/${bbox[3]}`)
+    const coOrds = {
```

Comment:
I'd probably extract this out to a `getCoords(bbox)` method to remove code duplication.